src/index.html

```diff
@@ -1,12 +1,530 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a class="logo" href="#HOME">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#APPLE">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#SAMSUNG">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link "
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#devices">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#skreen">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#pics">
+              фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#videos">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="catalog">
+
+        <div class="card-block catalog__item" data-qa="card">
+
+          <img
+            class="card-block__pic"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+
+          <div class="card-block__name">
+            <h2 class="card-block__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+          </div>
+
+          <div class="card-block__par">
+            <div class="card-block__parag">
+              Код товара: 195434
+            </div>
+          </div>
+
+          <div class="rewiev-container">
+
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="rewievs">
+              Отзывов: 5
+            </div>
+
+          </div>
+
+          <div class="card-block__price">
+            <div class="card-block__price-name">
+              Цена:
+            </div>
+
+            <div class="card-block__price-value">
+              69 999 грн
+            </div>
+
+          </div>
```

Comment:
Don't add blank lines after last element. Fix this in all places in your code
```suggestion
            </div>
          </div>
```